geode-core/src/distributedTest/java/org/apache/geode/internal/cache/partitioned/PersistentPartitionedRegionDistributedTest.java

```diff
@@ -1620,7 +1608,7 @@ public void recoveryFinished(Region region) {
 
     regionFactory.create(partitionedRegionName);
 
-    recoveryDone.await(2, MINUTES);
+    recoveryDone.await();
```

Comment:
CountDownLatch await() is infinite by default. You might want to change to use GeodeAwaitility timeout:
```
private static final long TIMEOUT_MILLIS = GeodeAwaitility.getTimeout().getValueInMS();

recoverDone.await(TIMEOUT_MILLIS, TimeUnit.MILLISECONDS);
```

Reply:
Done